client/src/hooks/useCreateUser.js

```diff
@@ -106,93 +50,29 @@ export const useCreateUser = (
 
     if (currentStep === '') {
       setCurrentStep(stepsArray[0])
-      setNeedsEmail(false)
     }
   })
 
-  const stepForward = () => {
-    const currentIndex = steps.indexOf(currentStep)
-    if (currentIndex < steps.length) {
-      setCurrentStep(steps[currentIndex + 1])
-    }
-  }
-
-  const stepBack = () => {
-    const currentIndex = steps.indexOf(currentStep)
-    if (currentIndex > 0) {
-      setCurrentStep(steps[currentIndex - 1])
-    }
-  }
-
-  const callCreateUser = async (
-    orcid,
-    accessToken,
-    refreshToken,
-    loginEmail,
-    loginGrants
-  ) => {
-    const [tokenRequest, userRequest] = await api.user.create(
-      orcid,
-      accessToken,
-      refreshToken,
-      loginEmail,
-      loginGrants
-    )
-
-    if (tokenRequest.status === 401) {
-      setNeedsEmail(true)
-      return { error: tokenRequest }
-    }
+  const createAndLoginUser = async () => {
+    const createUserRequest = await api.users.create(newUser)
 
-    if (!tokenRequest.isOk) {
-      console.log('error', tokenRequest)
-      return { error: tokenRequest }
+    if (createUserRequest.status === 401) {
+      return { error: createUserRequest }
     }
 
-    const { token } = tokenRequest.response
-
-    if (!userRequest.status === 200) {
-      console.log('error', userRequest)
-      return { error: userRequest }
+    if (!createUserRequest.isOk) {
+      // TODO::SENTRY
+      console.log('error', createUserRequest)
```

Comment:
Did you forget about this?

Reply:
we need a separate PR to trigger the explicit sentry calls - basically states that shouldnt happen we should know about them if they happen a lot